apps/users-service/src/controllers/users.ts

```diff
@@ -0,0 +1,84 @@
+import Boom from '@hapi/boom';
+import aws from 'aws-sdk';
+import { generate } from 'shortid';
```

Comment:
> ShortId does not generate cryptographically secure ids, so don't rely on it to make IDs which are impossible to guess.

Given that these could be used to impersonate a user, we should probably use e.g. UUID v4 from the `uuid` package.